eng/Versions.props

```diff
@@ -4,6 +4,7 @@
     <ProductVersion>5.0.0</ProductVersion>
     <MajorVersion>5</MajorVersion>
     <MinorVersion>0</MinorVersion>
+    <ServiceVersion>0</ServiceVersion>
```

Comment:
Core-Setup calls this `PatchVersion`: https://github.com/dotnet/core-setup/blob/master/eng/Versions.props#L6. We should use the same property name.